js/repl/Repl.js

```diff
@@ -94,7 +89,7 @@ function toCamelCase(str) {
     });
 }
 
-class Repl extends React.Component {
+class Repl extends React.Component<Props, State> {
   props: Props;
```

Comment:
Nit: we can remove these, since we're now declaring `Props` and `State` in L92.

Reply:
Thanks! Removed redundant ones.